tests/std/tests/P0466R5_layout_compatibility_and_pointer_interconvertibility_traits/test.cpp

```diff
@@ -55,21 +60,34 @@ constexpr bool test() {
         enum class E5 { zero, fortytwo = 42 };
 
         ASSERT(is_layout_compatible_v<int, int>);
+        ASSERT(is_layout_compatible_v<const void, void>);
+        ASSERT(is_layout_compatible_v<S1, volatile S2>);
+        ASSERT(is_layout_compatible_v<S1, S2>);
+        ASSERT(is_layout_compatible_v<S4, S4>);
+        ASSERT(is_layout_compatible_v<const volatile S4, S4>);
         ASSERT(is_layout_compatible_v<E1, E2>);
         ASSERT(is_layout_compatible_v<E3, E4>);
         ASSERT(is_layout_compatible_v<E5, E1>);
         ASSERT(is_layout_compatible_v<const E1, E2>);
         ASSERT(is_layout_compatible_v<volatile E3, const E4>);
-        ASSERT(is_layout_compatible_v<S1, volatile S2>);
-        ASSERT(is_layout_compatible_v<S1, S2>);
-        ASSERT(is_layout_compatible_v<S4, S4>);
-        ASSERT(is_layout_compatible_v<const volatile S4, S4>);
+        ASSERT(is_layout_compatible_v<int[], int[]>);
+        ASSERT(is_layout_compatible_v<int[3], int[3]>);
+
+        // TRANSITION, VSO-1269781
+        // ASSERT(is_layout_compatible_v<const int[], int[]>);
+        // ASSERT(is_layout_compatible_v<const int[3], int[3]>);
+        // ASSERT(is_layout_compatible_v<int[], volatile int[]>);

```

Comment:
VSO-1269781 is an MSVC bug, there's no reason to hide these particular cases from Clang and EDG (more than they already are).
```suggestion
#if defined(__clang__) || defined(__EDG__) // TRANSITION, VSO-1269781
        ASSERT(is_layout_compatible_v<const int[], int[]>);
        ASSERT(is_layout_compatible_v<const int[3], int[3]>);
        ASSERT(is_layout_compatible_v<int[], volatile int[]>);
#endif // TRANSITION, VSO-1269781
```